src/GitHub.Exports/Helpers/ICopyable.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace GitHub.Helpers
+{
+    public interface ICopyable<in T>
```

Comment:
Oh yeah, anything that wants to get tracked by this collection needs to implement this interface, but this PR only contains the collection and test interfaces, and this is mocked in the tests. Implementations are on subsequent PRs :smile: